sonic-slave-bullseye/Dockerfile.j2

```diff
@@ -426,6 +426,37 @@ RUN apt-get update && apt-get install -y eatmydata && eatmydata apt-get install
         default-jdk \
         libgoogle-gson-java
 
+{%- if CONFIGURED_PLATFORM == "vs" %}
```

Comment:
can we just install them by default in the slave container?

Reply:
No, there are some packages install failure on marvell platform

Reply:
that's fine, then why not say != marvel?

Reply:
IMHO, I didn't test all platforms, so `== vs` is safer than `!= marvel`.
But I fix it by following your suggestion in the latest commit.